.github/workflows/stale_issue.yml

```diff
@@ -44,3 +47,4 @@ jobs:
         loglevel: DEBUG
         # Set dry-run to true to not perform label or close actions.
         dry-run: false
+id-token
```

Comment:
is this a typo?

I don't see `id-token` documented as something you can put at the root of a workflow??
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

Reply:
Yes, it's a typo. Removed